admin/static/js/pages/prereg-admin-page.js

```diff
@@ -0,0 +1,254 @@
+'use strict';
+
+var $osf = require('js/osfHelpers');
+var ko = require('knockout');
+var $ = require('jquery');
+
+var drafts;
+
+ko.bindingHandlers.enterkey = {
+    init: function (element, valueAccessor, allBindings, viewModel) {
+        var callback = valueAccessor();
+        $(element).keypress(function (event) {
+            var keyCode = (event.which ? event.which : event.keyCode);
+            if (keyCode === 13) {
+                callback.call(viewModel);
+                return false;
+            }
+            return true;
+        });
+    }
+};
+
+/**
+* Assignee column.
+*
+* @param  {List} reviewers  The prereg reviewers who can be assigned
+*/
+var Assignee = function(reviewers) {
+    var self = this;
+    self.edit = ko.observable(false);
+
+    if (reviewers[0] !== 'none') {
+        reviewers.unshift('none'); 
+    }
+    
+    self.reviewers = ko.observableArray(reviewers);
+    self.assignee = ko.observable('none');
+};
+
+Assignee.prototype.editItem = function() {
+    var self = this;
+    self.assignee.edit(!self.editing);
+    self.editing(!self.editing);
+    
+};
+
+/**
+* Proof of publication column.
+*/
+var ProofOfPub = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.proofOfPub = ko.observable('Published Article Not Yet Submitted');
+    self.proofOfPubList = ko.observableArray(['Published Article Not Yet Submitted', 'Published Article Submitted', 'Published Article Under Review', 'Published Article Approved', 'Published Article Rejected']);
+};
+
+ProofOfPub.prototype.editItem = function() {
+    var self = this;
+    self.proofOfPub.edit(!self.editing);
+    self.editing(!self.editing);
+};
+
+/**
+* Payment sent column.
+*/
+var PaymentSent = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.paymentSent = ko.observable('no');
+    self.paymentSentList = ko.observableArray(['Yes', 'No']);
+};
+
+PaymentSent.prototype.editItem = function() {
+    var self = this;
+    self.paymentSent.edit(!self.editing);
+    self.editing(!self.editing);
+};
+
+/**
+* Notes column.
+*/
+var Notes = function() {
+    var self = this;
+    self.edit = ko.observable(false);
+    self.notes = ko.observable('none');
+};
+
+Notes.prototype.enlargeIcon = function(data, event) {
+    var icon = event.currentTarget;
+    $(icon).addClass("fa-2x");
+};
+
+Notes.prototype.shrinkIcon = function(data, event) {
+    var icon = event.currentTarget;
+    $(icon).removeClass("fa-2x");
```

Comment:
@leb2dg the Knockout way to do this would be:
```
<... data-bind="css: {hover: 'fa-2x'}, event: {hover: toggleHover}" >
```
where toggle hover toggles the state of a boolean observable on the view model.

And if you want to re-use this pattern of UI elements that change on hover, you might consider using the [oop module](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/oop.js) to create a mixin class to contain the duplicated logic (e.g. [ChangeMessageMixin] (https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/changeMessage.js))